src/pages/SingleArticle.jsx

```diff
@@ -34,6 +35,7 @@ export const SingleArticle = () => {
 					<br />
 					<LiaComments /> <span>{singleArticle.comment_count}</span>
 				</article>
+        <Comments article_id = {singleArticle.article_id}/>
 				<Link to={"/articles"}>
 					<button>Back to articles</button>
```

Comment:
doesnt make sense to have a Link (which is really just an anchor tag) wrapping a button! Not a good option for accessibility - you can always style the link to look like a button